pasystem/pasystem-tool/tool/pom.xml

```diff
@@ -71,4 +71,44 @@
 
   </dependencies>
 
+
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-assembly-plugin</artifactId>
+        <executions>
+          <execution>
+            <configuration>
+              <descriptor>src/template-zip-assembly.xml</descriptor>
+              <finalName>pasystem-samples</finalName>
+              <appendAssemblyId>false</appendAssemblyId>
+              <outputDirectory>${project.build.directory}/${project.build.finalName}</outputDirectory>
+            </configuration>
+            <phase>package</phase>
+            <goals>
+              <goal>single</goal>
+            </goals>
+          </execution>
+        </executions>
+      </plugin>
+      <plugin>
+        <artifactId>maven-war-plugin</artifactId>
+        <version>2.4</version>
```

Comment:
Why are you downgrading the war plugin here (we're generally using 2.6)? And I think the configurations you're supplying is the default anyway. 

https://maven.apache.org/plugins/maven-war-plugin/war-mojo.html#failOnMissingWebXml